src/cli.ts

```diff
@@ -9,7 +9,11 @@ import { transformSync } from "@babel/core";
 import { IModImplementation } from "./api/ModImplementation.js";
 // import { addCode } from "./api/RuntimeGenerators.js";
 
-if (process.argv.length != 5) {
+if (process.argv[5] != undefined && process.argv[5] !== "--entrypoint") {
+    console.error(`Usage:\n\t${myPackageName} <input file> <target client mod> <output file> --entrypoint\nExample:\n\t${myPackageName} ./index.js BetterDiscord ./dist/index.js --entrypoint`);
+    process.exit(1);
+}
+if (process.argv.length < 5) { // TODO: make a proper option parser
```

Comment:
You think so?

Reply:
use `yargs` for proper option parser. Don't reinvent the wheel

Reply:
I don't want to introduce more unnecessary packages